bootstrap

```diff
@@ -42,4 +42,9 @@ srcdir=`dirname ${0}`
 abs_srcdir=`pwd`
 abs_top_srcdir="${abs_srcdir}"
 
+# install Debian flavor dependencies
+if [ -f "Aptfile" ] && $(apropos "package manager" | grep -wq dpkg) ; then
```

Comment:
Should we just test for `apt-get` here, similar to what we did in https://github.com/openthread/borderrouter/blob/master/script/bootstrap?